### DISCUSSION TITLE
Can't access the Starred List on the Github App after change in account name

### DISCUSSION THREAD

**Discussion by acse-sd922** [community/community#38334] [Other Feature Feedback, Questions, & Ideas]
Like the title. The app was trying to fetch the starred list from my previous account name. The situation was resolved by logging out of and back in to the app.

**Comment by jfullstackdev** [comment]
that's one side effects of changing username, that's why there is the warning, I did that maybe 3x in the past and the effects could not even be predicted, it's not only in Lists but also others like retired repo names, broken links etc.

**Comment by heart-king** [comment]
Ashanlia

**Reply by Suteesayung** [reply]
https://luckygame777.com

**Comment by IldIko19890530** [comment]
Nice

**Reply by Eagleeye21xxx** [reply]
> 864e3bab3f353fb1

**Comment by FAHADHARTY** [comment]
Styles

**Comment by 1alexbc1** [comment]
> that's one side effects of changing username, that's why there is the warning, I did that maybe 3x in the past and the effects could not even be predicted, it's not only in Lists but also others like retired repo names, broken links etc.
